script.js

```diff
@@ -10,3 +10,215 @@ document.querySelectorAll('.nav-link').forEach((n) => n.addEventListener('click'
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 5,
+    title: 'Multi-Post Stories Gain+Glory Gain+Glory',
+    snapshot: './Images/assets/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    supdescription: 'Keeping track of hundreds of components',
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+  {
+    id: 6,
+    title: 'Multi-Post Stories Gain+Glory',
+    snapshot: './Images/assets/Snapshoot-Portfolio.svg',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry",
+    skills: ['Ruby on Rails', 'CSS', 'JavaScript', 'HTML'],
+    supdescription: 'Keeping track of hundreds of components',
+    projectInfo: { company: 'Microverse', role: 'Back END Dev', year: 2015 },
+    seeLive: 'https://github.com/newtonah0/Mobile-Portfolio',
+    seeSource: 'https://github.com/newtonah0/Mobile-Portfolio',
+  },
+
+];
+
+projects.forEach((project) => {
+  const bodySection = document.querySelector('.project-cards');
+  const section = document.createElement('section');
+  section.className = 'projects';
+  section.innerHTML = `<div class=""project-cards">
+  <img class="project-card" src=${project.snapshot} alt="Snap image for background">
+  </div>
+<div class="project-card_details">
+  <h2 class="title">${project.title}</h2>
+  
+  <div class="stacks">
+  <button class="stacks1" type="button" name="button">${project.skills[0]}</button>
+  <button class="stacks1" type="button" name="button">${project.skills[1]}</button>
+  <button class="stacks1" type="button" name="button">${project.skills[2]}</button>
+  <button class="stacks1" type="button" name="button">${project.skills[3]}</button>
+  </div>
+  
+  <div class="action-btn-container">
+  <button class="green-button" type="button" id="btn-${project.id}" name="button">See project</button>
+  
+  </div>
+</div>
+`;
+
+  bodySection.append(section);
+});
+const body = document.querySelector('section');
```

Comment:
- [ ] Good job so far by implementing some changes requested by the previous reviewer But I think there are a few things you need to fix in your design like different element positions (Buttons.. etc). Please make sure to stick to the Figma-chosen design as much as possible. You can check the element below to see the differences ⬇️ 

##### Figma design
![windows](https://github.com/newtonah0/Mobile-Portfolio/assets/6615394/e32e8958-d89b-4531-8d40-9eaf00d0de87)


##### Video with your design
https://www.loom.com/share/18c4eb0fd4c0420faf7a7ec750939443?sid=e93334e3-9f63-4a0a-9f82-05ca7ef634b5